packages/core/src/external/quest/schema/response.ts

```diff
@@ -0,0 +1,514 @@
+];
+
+export function isResponseHeader(data: object): data is ResponseHeader {
+  return responseHeaderSchema.safeParse(data).success;
+}
+
+export const responseDetailSchema = z.object({
+  dateOfService: z.date(),
+  accessionNumber: z.string(),
+  requisitionNumber: z.string().optional(),
+  labCode: z.string(),
+  dateCollected: z.date().optional(),
+  patientId: z.string(),
+  patientFirstName: z.string().optional(),
+  patientMiddleName: z.string().optional(),
+  patientLastName: z.string().optional(),
+  addressLine1: z.string().optional(),
+  addressLine2: z.string().optional(),
+  city: z.string().optional(),
+  state: z.string().optional(),
+  zipCode: z.string().optional(),
+  phoneNumber: z.string().optional(),
+  dateOfBirth: z.string().optional(),
+  patientAge: z.string().optional(),
+  gender: z.string().optional(),
+  socialSecurityNumber: z.string().optional(),
+  questBillingIdentifier: z.string().optional(),
+  memberNumber: z.string().optional(),
+  medicaidId: z.string().optional(),
+  medicareId: z.string().optional(),
+  orderingAccountNumber: z.string().optional(),
+  orderingAccountName: z.string().optional(),
+  orderingAccountAddressLine1: z.string().optional(),
+  orderingAccountAddressLine2: z.string().optional(),
+  orderingAccountCity: z.string().optional(),
+  orderingAccountState: z.string().optional(),
+  orderingAccountZipCode: z.string().optional(),
+  orderingAccountPhoneNumber: z.string().optional(),
+  physicianName: z.string().optional(),
+  physicianUpin: z.string().optional(),
+  physicianNpi: z.string().optional(),
+  diagnosisCode1: z.string().optional(),
+  diagnosisCode2: z.string().optional(),
+  diagnosisCode3: z.string().optional(),
+  diagnosisCode4: z.string().optional(),
+  diagnosisCode5: z.string().optional(),
+  diagnosisCode6: z.string().optional(),
+  diagnosisCode7: z.string().optional(),
+  diagnosisCode8: z.string().optional(),
+  diagnosisCode9: z.string().optional(),
+  diagnosisCode10: z.string().optional(),
+  localProfileCode: z.string().optional(),
+  standardProfileCode: z.string().optional(),
+  profileName: z.string().optional(),
+  localOrderCode: z.string(),
+  standardOrderCode: z.string().optional(),
+  orderName: z.string().optional(),
+  loincCode: z.string().optional(),
+  localResultCode: z.string(),
+  resultName: z.string().optional(),
+  resultValue: z.string().optional(),
+  resultUnits: z.string().optional(),
+  referenceRangeLow: z.string().optional(),
+  referenceRangeHigh: z.string().optional(),
+  referenceRangeAlpha: z.string().optional(),
+  abnormalFlag: z.string().optional(),
+  cptCode: z.string().optional(),
+  resultComments: z.string().optional(),
+});
+
+export type ResponseDetail = z.infer<typeof responseDetailSchema>;
+
+export function isResponseDetail(data: object): data is ResponseDetail {
+  return responseDetailSchema.safeParse(data).success;
+}
+
+export const responseDetailRow: IncomingFileRowSchema<ResponseDetail> = [
+  {
+    field: 1,
```

Comment:
Do we need `field`? Can't we use the array position?

Reply:
It's possible to have a skipped index, and this also makes it easier to verify the spec against the code.